### ISSUE TITLE
The confusion about the design of the loss function

### ISSUE THREAD

**Issue by Lucxin** [amjltc295/Free-Form-Video-Inpainting#36]
Hi, 

I have some confusion about the loss design. 

As the mentioned by Yu et al., SN-PatchGAN has encoded the patch-level information, so they omit the perceptual loss.
In your paper, except Temporal-PatchGAN, perceptual loss and style loss are also adopted to train the network. I want know why you add the perceptual and style loss.

Look forward to your answer. Thanks a lot.
Best wishes.

**Comment by amjltc295** [comment]
Those losses are added to stabilize training in the pre-training stage. You could also try to train without them.
